gulpfile.js

```diff
@@ -433,24 +499,26 @@ gulp.task('release', ['apps', 'clean-release'], function () {
             }
         }
     });
-
+        
     var platforms = getPlatforms(true);
     console.log('Packing release.');
 
+    var merged = mergeStream();
+    
     if (platforms.indexOf('chromeos') !== -1) {
         release_chromeos();
```

Comment:
Just out of curiosity. Why didn't you use `merge.add` when calling `release_chromeos()`?

Reply:
I think you're true. I didn't see the "gulp" word in the method, and I thinked it didn't use gulp. But if it uses an async stream, we must add it with the `merge.add`.